dask/dataframe/tests/test_rolling.py

```diff
@@ -166,6 +166,20 @@ def test_rolling_methods(method, args, window, center, check_less_precise):
     )
 
 
+@pytest.mark.parametrize("window", [1, 2, 4, 5])
+@pytest.mark.parametrize("center", [True, False])
+def test_rolling_cov(window, center):
+    # DataFrame
+    prolling = df.drop("a", 1).rolling(window, center=center)
+    drolling = ddf.drop("a", 1).rolling(window, center=center)
+    assert_eq(getattr(prolling, "cov")(), getattr(drolling, "cov")())
```

Comment:
Since you're not parametrizing on the method anymore, you can just call `cov` directly:

```suggestion
    assert_eq(prolling.cov(), drolling.cov())
```